src/Microsoft.Azure.WebJobs.Host.Storage/JobHostInternalStorageOptions.cs

```diff
@@ -40,5 +40,7 @@ public DistributedLockManagerContainerProvider(IOptions<JobHostInternalStorageOp
     public class JobHostInternalStorageOptions
     {
         public string InternalSasBlobContainer { get; set; }
+
+        public string InternalContainerName { get; set; }
```

Comment:
This new public member isn't used anywhere in the WebJobs SDK. This public contract implies that if this is set, we'll use this container across the board rather than the default "azure-webjobs-hosts", which isn't the case. E.g. if this were set, it should result in InternalContainer above for the lock manager using this option.

Reply:
Actually I see in the code below the comment "$$$ get rid of LegacyConfig" where this type is used. So this was intended to be temporary all along. Rather than expanding this, perhaps we just remove this and in Functions Host you just use the default container name? Not sure we need to make this configurable.

Reply:
That would be the simplest path if we don't need a configurable container. When I tested this override container, only the Lock Manager uses it, FunctionsSyncManager, secrets, etc. all use the default container. 

If we did want to support it, then we wouldn't be able to incorporate it as part of DistributedLockManagerContainerProvider since we need access to the StorageAccountOptions (internal object). We _could_ implement as part of the Create() method for the Lock Manager.

Reply:
Talked with Fabio. InternalContainerName would be added behavior - I'm going to just take it out, since the new concrete lock implementation in the host already supports the sas container override.